environs/instances/image_test.go

```diff
@@ -371,6 +371,53 @@ func (s *imageSuite) TestFindInstanceSpec(c *gc.C) {
 	}
 }
 
+func (s *imageSuite) TestFindInstanceSpecShouldChooseNonSEV(c *gc.C) {
```

Comment:
The struct we use is defined in `environs/instances` so you'd think we'd have a test for `FindInstanceSpec` in that package but seems not. Each provider seems to do their own tests so this seems ok.